webapp/components/features/ReportsTable/ReportsTable.vue

```diff
@@ -24,11 +24,8 @@
       </tr>
     </thead>
     <template v-for="report in reports">
-      <report-row
-        :key="report.resource.id"
-        :report="report"
-        @confirm-report="$emit('confirm', report)"
-      />
+      <!-- should be ':key="report.resource.id"' for having one element for every resource, but this crashes at the moment, because the 'reports' query returns multiple reports on the same resource! I will create an issue -->
```

Comment:
> [<img alt="mattwr18" height="40" width="40" align="left" src="https://avatars1.githubusercontent.com/u/38261864?s=88&v=4">](https://github.com/mattwr18) **Authored by [mattwr18](https://github.com/mattwr18)**
_<time datetime="2020-02-20T10:34:02Z" title="Thursday, February 20th 2020, 11:34:02 am +01:00">Feb 20, 2020</time>_

---
> **Outdated (history rewrite)** - original diff
---
```diff
@@ -24,11 +24,8 @@
       </tr>
     </thead>
     <template v-for="report in reports">
-      <report-row
-        :key="report.resource.id"
-        :report="report"
-        @confirm-report="$emit('confirm', report)"
-      />
+      <!-- should be ':key="report.resource.id"' for having one element for every resource, but this crashes at the moment, because the 'reports' query returns multiple reports on the same resource! I will create an issue -->
```


```suggestion
      
```
I believe this is not an issue, as `all` reports does not mean `unique` reports for a resource. Also, why should it use the resource id? the report ids are unique.